diff --git a/node_modules/react-native-windows/local-cli/runWindows/utils/msbuildtools.js b/node_modules/react-native-windows/local-cli/runWindows/utils/msbuildtools.js
index 63635c631..802f7f9a8 100644
--- a/node_modules/react-native-windows/local-cli/runWindows/utils/msbuildtools.js
+++ b/node_modules/react-native-windows/local-cli/runWindows/utils/msbuildtools.js
@@ -174,10 +174,12 @@ function checkMSBuildVersion(version, buildArch, verbose) {
   const requires16 = [
     'Microsoft.Component.MSBuild',
     getVC141Component(version, buildArch),
-    'Microsoft.VisualStudio.ComponentGroup.UWP.VC.v141',
   ];
   const requires15 = [
     'Microsoft.Component.MSBuild',
     getVC141Component(version, buildArch),
-    'Microsoft.VisualStudio.ComponentGroup.UWP.VC',
   ];

   const requires = version === '16.0' ? requires16 : requires15;
