diff --git a/node_modules/react-native-windows/Libraries/Network/XMLHttpRequest.js b/node_modules/react-native-windows/Libraries/Network/XMLHttpRequest.js
index f0f507c..8331654 100644
--- a/node_modules/react-native-windows/Libraries/Network/XMLHttpRequest.js
+++ b/node_modules/react-native-windows/Libraries/Network/XMLHttpRequest.js
@@ -468,8 +468,7 @@ class XMLHttpRequest extends EventTarget(...XHR_EVENTS) {
       throw new Error('Request has already been sent');
     }
     this._sent = true;
-    const incrementalEvents =
-      this._incrementalEvents || !!this.onreadystatechange || !!this.onprogress;
+    const incrementalEvents = false;
 
     this._subscriptions.push(
       RCTNetworking.addListener('didSendNetworkData', args =>
