diff --git a/node_modules/wd/lib/webdriver.js b/node_modules/wd/lib/webdriver.js
index 33d9c85..0e63879 100644
--- a/node_modules/wd/lib/webdriver.js
+++ b/node_modules/wd/lib/webdriver.js
@@ -124,6 +124,7 @@ Webdriver.prototype._init = function() {
 
     // retrieving session
     try{
+      console.log('Driver resp:', data);
       jsonData = JSON.parse(data);
       _this.sessionID = jsonData.value ? jsonData.value.sessionId : false;
       _this.w3c = false;
